### ISSUE TITLE
Google Home Support [Suggestion]

### ISSUE THREAD

**Issue by PhantomNimbi** [vfsfitvnm/ViMusic#896]
### Steps to reproduce the bug

On Google Home device say play Maroon 5 Sugar in ViMusic

### Expected behavior

Should play the specified song or playlist on your connected smart speaker.

### Actual behavior

Currently nothing since it's not implemented.

### ViMusic version

"0.5.4"

### Android version

"Android 11"

### Additional information

This is not a bug report but instead a feature request. There was no template for feature requests so I simply used what was available.

Basically though what I'm requesting is Google Home Support for use with connected smart speakers such as the Google Nest Mini and so on.

I would love to be able to cast this to my Nest Mini. Honestly just having the option for Chromecast would be a massive improvement in my opinion.
